src/PanoViewer/PanoViewer.js

```diff
@@ -378,70 +413,87 @@ export default class PanoViewer extends Component {
 	}
 
 	/**
-	 * When set true, enables zoom with the wheel and Pinch gesture
-	 * @ko true 로 설정 시 휠 및 집기 동작으로 확대 / 축소 할 수 있습니다.
+	 * When set true, enables zoom with the wheel or pinch gesture. However, in the case of touch, pinch works only when the touchDirection setting is {@link eg.view360.PanoViewer.TOUCH_DIRECTION.ALL}.
+	 * @ko true 로 설정 시 휠 혹은 집기 동작으로 확대/축소 할 수 있습니다. false 설정 시 확대/축소 기능을 비활성화 합니다. 단, 터치인 경우 touchDirection 설정이 {@link eg.view360.PanoViewer.TOUCH_DIRECTION.ALL} 인 경우에만 pinch 가 동작합니다.
 	 * @method eg.view360.PanoViewer#setUseZoom
 	 * @param {Boolean} useZoom
+	 * @return {eg.view360.PanoViewer} PanoViewer instance<ko>PanoViewer 인스턴스</ko>
 	 */
 	setUseZoom(useZoom) {
 		if (typeof useZoom !== "boolean") {
-			return;
+			return this;
 		}
 
 		this._yawPitchControl.option("useZoom", useZoom);
+		return this;
```

Comment:
You can make the line of code shorter by applying the following changes.
```js
typeof useZoom === "boolean" && this._yawPitchControl.option("useZoom", useZoom);
return this;
```

Reply:
@happyhj Ok, I'll take it